tools/base/src/Base.ts

```diff
@@ -183,7 +184,9 @@ export function SpectrumMixin<T extends Constructor<ReactiveElement>>(
     return SpectrumMixinElement;
 }
 
-export class SpectrumElement extends SpectrumMixin(LitElement) {}
+export class SpectrumElement extends SpectrumMixin(LitElement) {
+    static VERSION = version;
```

Comment:
Do we want this on any files built from the mixin, as well?

How do you think this location will effect projects that leverage SpectrumElement as a base class but aren't SWC?

Reply:
Please correct me If I am wrong ,upon checking in the repository for [projects](https://git.corp.adobe.com/CCX/cloud-shared-components/tree/develop/packages/projects) I couldn't find an implementation which uses SpectrumElement as a base class.  I think best person to answer this would be @rickharris related to projects query.

Reply:
Personally I don't have enough info to know where the right place to add the version is. Hopefully the answer to 

> How do you think this location will effect projects that leverage SpectrumElement as a base class but aren't SWC?

is "minorly to not at all", but I really am not confident about that at all. I suppose that in that case it would still represent the version of `SpectrumElement` being used, and an extending component that's not SWC could override it they needed to.

Reply:
We can probably work that out over time.